### ISSUE TITLE
Coordinate system when loading mesh

### ISSUE THREAD

**Issue by clemense** [mikedh/trimesh#183]
Hi,
When loading an obj-mesh in trimesh vs. blender, the mesh is differently oriented (z-axis up in blender, y-axis up in trimesh). Is there a setting for the loading the triangle coordinates that affects this?

**Comment by mikedh** [comment]
Interesting, are you sure blender isn't adding a transform in somewhere? As far as I know, the trimesh OBJ loader tries to maintain the coordinates in the file and doesn't swap columns or anything like that. On first inspection it looks OK:

```
In [2]: c = trimesh.primitives.Cylinder(height=10)

In [3]: c.export('hi.obj')

In [4]: d = trimesh.load('hi.obj')

In [6]: c.bounds
Out[6]:
array([[-0.99871651, -0.99486932, -5.        ],
       [ 0.99871651,  1.        ,  5.        ]])

In [7]: d.bounds
Out[7]:
array([[-0.99871651, -0.99486932, -5.        ],
       [ 0.99871651,  1.        ,  5.        ]])
```

As a quick check, if you load the OBJ in question into meshlab, and click `render -> show axis`, do the axis shown agree with blender, or with trimesh?

**Comment by clemense** [comment]
You are right: Meshlab agrees with trimesh, while Blender has some 90degree rotation.

**Comment by mikedh** [comment]
Glad you figured it out! Also glad it's their bizarre design choices not mine 😁
